src/libcore/mem.rs

```diff
@@ -1152,15 +1152,6 @@ impl<T> MaybeUninit<T> {
         MaybeUninit { uninit: () }
     }
 
-    /// Deprecated before stabilization.
-    #[unstable(feature = "maybe_uninit", issue = "53491")]
-    #[inline(always)]
-    // FIXME: still used by stdsimd
```

Comment:
Was this done?

Reply:
Yes, this landed with https://github.com/rust-lang/rust/pull/59556